joblib/numpy_pickle.py

```diff
@@ -414,9 +413,18 @@ def dump(value, filename, compress=0, protocol=None, cache_size=None):
                 '(compress method, compress level), you passed {}'
                 .format(compress))
         compress_method, compress_level = compress
+    elif isinstance(compress, _basestring):
+        compress_method = compress
+        compress_level = 3  # Use default compress level
+        compress = (compress_method, compress_level)
     else:
         compress_level = compress
 
+    if compress_method == 'lz4' and lz4 is None:
+        raise ValueError('LZ4 in not installed. Consider installing it '
+                         'from PyPI: https://pypi.python.org/pypi/lz4 '
+                         'using: "pip install lz4"')
```

Comment:
Let's stay neutral w.r.t. the installation method (pip, conda, debian...), it's up to the user to decide.

```python
raise ValueError('LZ4 in not installed. Consider installing it: '
                 'http://python-lz4.readthedocs.io/')
```